### PR TITLE
Commented out 'nova boot --poll' until proper fix

### PR DESCRIPTION
**Pull request by nicolasbock**
https://trello.com/c/m9a3MQYS/173-c6-nova-boot-for-manila-failed

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3273,12 +3273,16 @@ function oncontroller_manila_generic_driver_setup()
...
     if [ "`openstack server show --f value -c status manila-service`" != ACTIVE ]; then
-        fixed_net_id=`neutron net-show fixed -f value -c id`
-        timeout 10m nova boot --poll --flavor 100 --image manila-service-image \
-            --security-groups $sec_group,default \
-            --nic net-id=$fixed_net_id manila-service
-
-        [ $? != 0 ] && complain 43 "nova boot for manila failed"
+        # Removed boot --poll test until a proper fix is in place.
+        # https://trello.com/c/m9a3MQYS/173-c6-nova-boot-for-manila-failed
+        if test ${hacloud} -ne 1; then
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: We want to remove the `--poll` parameter from the command if we are in ha, not the full command

Comment (abel-navarro) [inline scripts/qa_crowbarsetup.sh]: when `$hacloud -eq 1` when also want `nova boot` but without `--poll`.

**Comment by aplanas** [comment]
Also we need to do smthing similar in https://github.com/nicolasbock/automation/blob/nova_boot_poll/scripts/qa_crowbarsetup.sh#L3453

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Ok, thanks for the comments. I will rework this patch. Just as a side-note, shouldn't we require bash here and not simply sh? We are using some bash in that script (like [[ ]]).

Comment (abel-navarro) [inline scripts/qa_crowbarsetup.sh]: I think you are right. Just checked one of the nodes where we usually run mkcloud and sh seems to be mapped to bash. I won't change it for now.
```
mkcloud2:~ # /bin/sh --version
GNU bash, version 3.2.57(1)-release (x86_64-suse-linux-gnu)
Copyright (C) 2007 Free Software Foundation, Inc.
```

Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Abel, you are right, often it is linked to bash, but you could also use dash, tcsh, or whatever else you like as the link target. Maybe that's not what you would do an SLES though :)

**Comment by aplanas** [comment]
+1

**Comment by abel-navarro** [comment]
+1

**Comment by toabctl** [comment]
Isn't that just hiding the problem that nova is not working in ha mode?

**Comment by aplanas** [comment]
@toabctl is `--poll` the part that have a problem with haproxy. I am patching the `.get` to really resolve the issue, but with this we unlock the gating and the tests in jenkins

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -3273,8 +3273,16 @@ function oncontroller_manila_generic_driver_setup()
     fi
 
     if [ "`openstack server show --f value -c status manila-service`" != ACTIVE ]; then
+        # Removed --poll argument until a proper fix is in place.
+        # https://trello.com/c/m9a3MQYS/173-c6-nova-boot-for-manila-failed
+        local extra_args="--poll"
```
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Changed the logic so that we don't die when hacloud is not set.

**Comment by aplanas** [comment]
`if [[ $hacloud = 1 ]] ; then`

**Comment by nicolasbock** [comment]
Ok, I'll make it more consistent.

**Comment by abel-navarro** [comment]
Let's please hold this PR because I think I found a fix changing haproxy options. PRs will come very soon.

**Comment by jdsn** [comment]
> Ok, I'll make it more consistent.

@nicolasbock please do the `[[ $hacloud = 1 ]]` changes as separate commit.
This follows the rule of single logical change per commit.

**Comment by aplanas** [comment]
IMHO we can close it if https://github.com/crowbar/barclamp-pacemaker/pull/222 is decided

**Comment by nicolasbock** [comment]
I agree, @jdsn this was meant simply as s stop gap measure until we have a more proper fix.

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -3446,7 +3452,16 @@ function oncontroller_testsetup()
     nova secgroup-add-rule default icmp -1 -1 0.0.0.0/0
     nova secgroup-add-rule default tcp 1 65535 0.0.0.0/0
     nova secgroup-add-rule default udp 1 65535 0.0.0.0/0
-    timeout 10m nova boot --poll --image $image_name --flavor $flavor --key_name testkey testvm | tee boot.out
+    # Removed --poll argument until a proper fix is in place.
+    # https://trello.com/c/m9a3MQYS/173-c6-nova-boot-for-manila-failed
+    local extra_args="--poll"
+    if [[ $hacloud = 1 ]] ; then
+        extra_args=""
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: Nitpicking warning:
This can be shortened to: `extra_args=`

**Comment by nicolasbock** [comment]
Should we close this one since we don't really need it anymore?

**Pull request closed without merging**